src/main/scala/dpla/ingestion3/enrichments/Spatial.scala

```diff
@@ -0,0 +1,172 @@
+ *
+ *  dc:spatial [
+ *    a edm:Place;
+ *    dpla:providedLabel "Los Angeles (Calif.)";
+ *    gn:countryCode "US";
+ *    geo:lat 3.405223e1;
+ *    geo:long -1.1824368e2;
+ *    skos:closeMatch <http://id.loc.gov/authorities/names/n79021240>;
+ *    skos:exactMatch <http://sws.geonames.org/5368361/>;
+ *    skos:prefLabel "Los Angeles, CA, United States"
+ *  ];
+ *
+ * Note that Ingestion 2 (specifically, the Audumbla::CoarseGeocode enrichment)
+ * was bailing on the assignment of parent features, probably due to the fact
+ * that MAPv4 specifies them as URIs, which is inconvenient for us when it comes
+ * time to serialize the records into JSON-LD and have it indexed effectively
+ * by Elasticsearch, where you want strings for the parent features (E.g.
+ * "Los Angeles County" or "California"), not URIs. Our users search for those
+ * strings!
+ *
+ * We have work to do to determine if MAPv4 is actually the right thing, and
+ * whether we need a revision; or if we want to jump through hoops to resolve
+ * parent features as part of the process to writing JSON for Elasticsearch to
+ * index.
+ */
+
+/* Case classes for JSON parser ... */
+case class Coords(lat: Double, lng: Double)
+
+case class Bounds(ne: Coords, sw: Coords)
+
+case class Geometry(center: Coords, bounds: Bounds)
+
+case class Feature(geometry: Geometry, woeType: Int, displayName: String)
+
+case class Interpretation(feature: Feature)
+
+case class Result(interpretations: List[Interpretation])
+
+object Spatial extends MappingUtils with DefaultVocabularies {
+
+  implicit val formats = DefaultFormats // Formats for json4s
+  val log = Logger.getLogger(getClass.getName)
+
+  def enrich(m: Model): Model = {
+
+    val spatialModel = m.filter(null, rdf.`type`, edm.Place)
+    if (spatialModel.isEmpty) return m
+
+    for {
+      s <- spatialModel.subjects
+      provLabels = m.filter(s, dpla.providedLabel, null)
+      pl <- provLabels
+    } {
+
+      /* We expect the provided label below not to need any special processing,
+       * like stripping newlines, etc. That should be handled by an earlier
+       * enrichment.
+       *
+       * If it's a multiline string, as we've encountered with Digital
+       * Commonwealth, I really don't know if we can handle it well. Twofishes
+       * will not do what you want with a string like "Dallas Dallas Texas
+       * United States North and Central America". This particular instance will
+       * work if the values are reversed (as "North and Central America United
+       * States Texas Dallas Dallas") but I wouldn't want to count on that in
+       * general.
+       *
+       * We need some research to determine how common this is. Ideally we'd
+       * have multiple dc:spatial statements instead of one statement with a
+       * multiline string as the object.
+       *
+       * Is this an issue with Krikri that won't happen with Ingestion 3?
+       */
+
+      try {
```

Comment:
Another place where we will eventually want to return a `Try` and make the caller deal with what to do when something fails.